src/main/resources/reference.conf

```diff
@@ -216,6 +216,55 @@ cassandra-query-journal {
   
   # For each `persistenceId` the events are delivered strictly by sequence number. If
   # a sequence number is missing the query is delayed up to the configured
+  # `wrong-sequence-number-order-timeout` and if the expected event is still not found
+  # the stream is completed with failure.
+  wrong-sequence-number-order-timeout = 30s
```

Comment:
something is wrong with the merge, please rebase this on top of my latest commit in cassandra-3.x